src/main/java/com/parasoft/findings/utils/common/logging/ILoggerHandler.java

```diff
@@ -0,0 +1,60 @@
+/*
```

Comment:
Why do we need `ILoggerHandler `and `ILoggerHandlerFactory `as interfaces when they are only implemented by one class? Are we planning to implement them with more classes?